Cargo.toml

```diff
@@ -54,6 +55,11 @@ bincode = "1.3.3"
 wasm-bindgen-test = "0.3"
 getrandom = { version = "0.2", features = ["js"] }
 
+[target.'cfg(all(target_os = "zkvm", target_vendor = "succinct"))'.dependencies]
+sp1-ecdsa = { path = "/home/adityabisht64/signatures/ecdsa", package = "ecdsa", features = ["verifying", "alloc"] }
```

Comment:
Point this to https://github.com/sp1-patches/signatures on the `patch-ecdsa-v0.16.9`